src/software/thunderscope/robot_communication.py

```diff
@@ -203,30 +203,17 @@ def __enter__(self):
             True,
         )
 
+        self.send_primitive_mcast_sender = PrimitiveSetProtoSender(
+            self.multicast_channel + "%" + self.interface, PRIMITIVE_PORT, True
+        )
```

Comment:
This is actually not used and seems like a duplicate of whats on line 218

Reply:
yeah I noticed this but I didn't want to remove it, but I'll remove it now